From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
From: Tom Levy <tomlevy93@gmail.com>
Date: Sat, 18 Jul 2020 06:02:38 +1200
Subject: Use two lines for problem name and points in scoreboard header

Having them on a single line can make the scoreboard very wide.
---
 webapp/templates/partials/scoreboard_table.html.twig | 1 +
 1 file changed, 1 insertion(+)

diff --git a/webapp/templates/partials/scoreboard_table.html.twig b/webapp/templates/partials/scoreboard_table.html.twig
index f427995b5..72d4dc763 100644
--- a/webapp/templates/partials/scoreboard_table.html.twig
+++ b/webapp/templates/partials/scoreboard_table.html.twig
@@ -88,6 +88,7 @@
                             <div class="circle" style="background: {{ problem.color }};"></div>
                         {% endif %}
                         {% if showPoints %}
+                            <br>
                             <span class='problempoints'>
                                 [{% if problem.points == 1 %}1 point{% else %}{{ problem.points }} points{% endif %}]
                             </span>
-- 
2.20.1

